### PR TITLE
Replace own commands with MiddlewareClientBundle Commands

### PR DESCRIPTION
**Pull request by DRvanR**
Depends on https://github.com/SURFnet/Stepup-Middleware-clientbundle/pull/18

- [ ] Update composer.json to point to develop after merging aforementioned PR

### CODE REVIEW

**Comment by DRvanR** [comment]
@rjkip

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Service/YubikeySecondFactor/ProofOfPossessionResult.php [hunk 1]
```diff
@@ -91,7 +132,7 @@ public function didOtpVerificationFail()
     /**
      * @return boolean
      */
-    public function didProofOfPossessionFail()
+    public function didProofOfPossessionCommandFail()
```
Comment (rjkip) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Service/YubikeySecondFactor/ProofOfPossessionResult.php]: Is it relevant to the consumer of the service that it was in fact the execution of a command that failed? Imo the consumer is merely asking the possession to be proven using whatever method the service sees fit.

**Comment by rjkip** [comment]
Nice `ProofOfPossessionResult` named constructors. :+1:  Do what you want with my question.

File: src/Surfnet/StepupSelfService/SelfServiceBundle/Service/YubikeySecondFactor/ProofOfPossessionResult.php (hunk 1 above)
Comment (DRvanR) [inline src/Surfnet/StepupSelfService/SelfServiceBundle/Service/YubikeySecondFactor/ProofOfPossessionResult.php]: It was/is the structure and apparent intent of the Result objects to communicate success/failure and if failed, why it failed. This in order to be able to provide correct feedback to the user (at least, that is what I gathered from it).
This name was changed in order to better reflect the fact that the Proof of Possession as given by the User was in fact successful, however there was a system error (see the named constructor and inner logic of the class). 
I'm not to keen on how these classes are implemented in general though...
Will add a chore to revisit them later as we have more pressing matters to tend to :smiley:

**Pull request merged**